vscode-dotnet-runtime-library/src/Utils/Debugging.ts

```diff
@@ -0,0 +1,23 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+/**
+ * A simple wrapper around console logging that can disable / enable all debugging or logging messages.
+ */
+export class Debugging {
+    static debugOn = false;
+
+    public static log(message : string)
+    {
+        if(Debugging.debugOn)
+        {
+            console.log(message);
+        }
+        else
+        {
+            ; // do nothing to appease ansync?
```

Comment:
```suggestion
            ; // do nothing to appease async?
```
Remove this else statement?